frontend/public/components/_toggle-play.scss

```diff
@@ -6,6 +6,10 @@
   width: 32px;
   padding: 0;
 
+  &--space-r {
```

Comment:
Nit:  I'm not a big fan of class names that describe what they do.  I would rather see a class name that semantically describes the content rather than the presentation.  Additionally, you could just put the margin right on the parent as the other context that uses ToggleButton [has an override that sets the margin to 0](https://github.com/openshift/console/blob/master/frontend/public/components/_sysevent-stream.scss#L60).

Reply:
Yeah, I was trying to avoid adding a new class just for this context since this pattern might be used elsewhere. One option would be to make the status message an optional property of the TogglePlay component itself so that we can include styling in that component.